task/05-regex-tasks.js

```diff
@@ -31,7 +31,7 @@
  * @return {RegExp}
  */
 function getRegexForGuid() {
-   throw new Error('Not implemented');
+   return /{[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}}/;
 }
```

Comment:
we don't need the task here, please remove move the changes to a separate branch and PR

Reply:
I don`t understand how to create a new branch without the label on the image. Every new branch has all my commits, but i expected that it has only commits connected for example with task #7. Can you explain me how to solve it?)
![qw](https://user-images.githubusercontent.com/59390725/86530041-fd8a0a00-bebd-11ea-9aa9-64cb496d0d8c.jpg)